log4j-core/src/main/java/org/apache/logging/log4j/core/util/WatchManager.java

```diff
@@ -112,15 +111,11 @@ public void run() {
                 final ConfigurationMonitor monitor = entry.getValue();
                 if (monitor.getWatcher().isModified()) {
                     final long lastModified = monitor.getWatcher().getLastModified();
-                    if (logger.isInfoEnabled()) {
-                        logger.info(
-                                "Source '{}' was modified on {} ({}), previous modification was on {} ({})",
-                                source,
-                                millisToString(lastModified),
-                                lastModified,
-                                millisToString(monitor.lastModifiedMillis),
-                                monitor.lastModifiedMillis);
-                    }
+                    logger.info(
+                            "Configuration source at {} was modified on {}, previous modification was on {}",
```

Comment:
There are several places in this PR where logging message is changed/improved. In those places, I'd really appreciate text parameters to be wrapped with backticks. For instance,

```suggestion
                            "Configuration source at `{}` was modified on `{}`, previous modification was on `{}`",
```